source/Octopus.Tentacle/Scripts/NoopScriptStateWorkspace.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Scripts
+{
+    public class NoopScriptStateWorkspace : IScriptStateWorkspace
+    {
+        public bool StateExists => false;
+        public ScriptState Create()
+        {
+            return new ScriptState(new ScriptTicket(string.Empty));
+        }
+
+        public void Save(ScriptState state)
+        {
+        }
+
+        public ScriptState Load()
+        {
+            return new ScriptState(new ScriptTicket(string.Empty));
```

Comment:
It feels dodgy to have an implementation that returns misleading values. It could be a trap someone falls into.

Could this be an in-memory implemented instead?